+srv/expServer.m

```diff
@@ -345,6 +360,29 @@ function calibrateWaterDelivery()
     save(rigHwFile, 'daqController', '-append');
   end
 
+    function viewCalibration()
+        
+        calib = rig.daqController.SignalGenerators(rewardId).Calibrations(end);
```

Comment:
I got an error because the calibration was missing.  I'll make a change to deal with this.